x-pack/plugins/security_solution/public/common/store/inputs/model.ts

```diff
@@ -64,16 +59,12 @@ export interface GlobalGenericQuery {
   invalidKqlQuery?: Error;
 }
 
-export interface GlobalGraphqlQuery extends GlobalGenericQuery {
```

Comment:
this was making a type angry, and we don't have graphql queries any longer so we can delete